### ISSUE TITLE
V5 Beta - LDAP User Sync: manager id is invalid

### ISSUE THREAD

**Issue by bricelabelle** [snipe/snipe-it#6781]
#### Please confirm you have done the following before posting your bug report:

- [ x] I have enabled debug mode 
- [ x] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
When running an LDAP user sync, I get an issue for some users that says: 

> 'username' was not imported. REASON: The selected manager id is invalid.

I get this for the V5.0 beta, but not in my production build 4.6.6. 

I'm wondering if it's related to the fact that whenever I click into a user, I get these errors:

> (1/2) ParseErrorsyntax error, unexpected end of file, expecting elseif (T_ELSEIF) or else (T_ELSE) or endif (T_ENDIF)
> --
> in d7f23508de36392844beca9682c366be4e5b4681.php line 597

> (2/2) ErrorExceptionsyntax error, unexpected end of file, expecting elseif (T_ELSEIF) or else (T_ELSE) or endif (T_ENDIF) (View: /var/www/html/resources/views/users/view.blade.php)
> --
> in d7f23508de36392844beca9682c366be4e5b4681.php line 597

**To Reproduce**
Run an LDAP sync. 

**Server (please complete the following information):**
Docker for MacOS using V5 Beta build. 

**Desktop (please complete the following information):**
MacOS
Safari

**Additional context**
- Is this a fresh install or an upgrade? 
This is a clean docker build that I imported my production database to.

**Comment by snipe** [comment]
Are you still seeing this issue? You shouldn't be seeing a parse error.

> 'username' was not imported. REASON: The selected manager id is invalid.

That's not 100% surprising - you have to run it twice (same as for a CSV import), since the manager doesn't exist yet.

**Comment by bricelabelle** [comment]
I updated to the latest develop build and I'm no longer seeing the parse errors. :) I can click on a user and see their details without a problem. I am still getting the manager id is invalid error, even after running the import again. I should have mentioned two things before:

1. I've already imported my previous prod database, including users, prior to running the LDAP sync.
2. We don't have managers assigned in our prod database so I'm not sure why it's even trying to sync that information during the LDAP sync. 

Could either of these be causing the issue?
